web/static/js/components/respondents/RespondentIndex.jsx

```diff
@@ -431,6 +472,7 @@ class RespondentIndex extends Component<Props, State> {
               const responses = respondentsFieldName.map((field) => {
                 return {
                   name: field,
+                  isNumeric: this.fieldIsNumeric(numericFields, field),
```

Comment:
Instead of adding `isNumeric` to each data, how about adding a `cellClassNames` property to `<RespondentRow>`.

```jsx
<RespondentRow cellClassNames={(fieldName) => classNames({
    'tdNowrap' : true, 
    'tdNumber' : this.fieldIsNumeric(numericFields, fieldName)})}
/>
```

That way it is clear that all cells of the same field will be styled in the same way.